libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -505,6 +505,62 @@ def CreateImageFromDisk(self,
     self.BlockOperation(response)
     return GoogleComputeImage(self.project_id, '', name)
 
+  def CreateImageFromGcsTarGz(
+      self,
+      gcs_uri: str,
+      name: Optional[str] = None) -> 'GoogleComputeImage':
```

Comment:
See my comment on https://github.com/google/cloud-forensics-utils/pull/192/files#r451488908, I'm not sure `= None` is needed here.

Reply:
looks like we will leave the None